frame/wormhole/backing/ethereum/src/lib.rs

```diff
@@ -578,10 +578,8 @@ pub mod pallet {
 				if is_ring { <Error<T>>::RingLockedNSBA } else { <Error<T>>::KtonLockedNSBA }
 			);
 			// // Checking redeemer have enough of balance to pay fee, make sure follow up transfer
-			// will success. ensure!(
-			// 	T::RingCurrency::usable_balance(redeemer) >= fee,
-			// 	<Error<T>>::FeeIns
-			// );
+			// will success.
```

Comment:
@xiaoch05 Any suggestion here?

Reply:
Perhaps the fee's system here is not yet fully realized. cc @AurevoirXavier

Reply:
Yep.
Let's leave it here for now.